From 909bc15ac5ed2742cd991f96cfe6100a24c14c26 Mon Sep 17 00:00:00 2001
Message-ID: <909bc15ac5ed2742cd991f96cfe6100a24c14c26.1737500778.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1737500778.git.sharlatanus@gmail.com>
References: <cover.1737500778.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 15 Jan 2025 07:52:42 +0000
Subject: [PATCH 10/27] gnu: python-czml3: Update to 2.3.0.

The project's maintainer has been changed, see
<https://github.com/poliastro/czml3/issues/112>.

* gnu/packages/astronomy.scm (python-czml3): Update to 2.3.0.
[source] <url>: Adjust it as project's owner has been changed.
[native-inputs]: Remove python-astropy-minimal, and python-pytest-mypy.
[propagated-inputs]: Remove python-attrs. Add python-numpy,
python-pydantic-2, python-strenum, and python-typing-extensions.
[home-page]: Adjust it as project's owner has been changed.
[license]: Likewise.

Change-Id: I71116eaf53ce87ff72a4f238362cd2669e6e9ddb
---
 gnu/packages/astronomy.scm | 23 +++++++++++++----------
 1 file changed, 13 insertions(+), 10 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index ff879490409..1cb57aa09fb 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2193,34 +2193,37 @@ (define-public python-crds
 (define-public python-czml3
   (package
     (name "python-czml3")
-    (version "1.0.2")
+    (version "2.3.0")
     (source
      (origin
        (method git-fetch)
        (uri (git-reference
-             (url "https://github.com/poliastro/czml3")
+             ;; Maintainer of the project has been changed, see
+             ;; <https://github.com/poliastro/czml3/issues/112>.
+             (url "https://github.com/Stoops-ML/czml3")
              (commit (string-append "v" version))))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "0gsdffn5w7j1aridvmx4xh6axvmbmqjwawacc8fnshpcyh4md5pn"))))
+        (base32 "1125239knl4w2njnrqvdc5qzjvq80zlf247z72za7qz6k0rnv3rw"))))
     (build-system pyproject-build-system)
     (native-inputs
-     (list python-astropy-minimal
-           python-pytest
-           python-pytest-mypy
+     (list python-pytest
            python-setuptools
            python-wheel))
     (propagated-inputs
-     (list python-attrs
-           python-dateutil
+     (list python-dateutil
+           python-numpy
+           python-pydantic-2
+           python-strenum
+           python-typing-extensions
            python-w3lib))
-    (home-page "https://github.com/poliastro/czml3")
+    (home-page "https://github.com/Stoops-ML/czml3")
     (synopsis "Python library to write CZML")
     (description
      "CZML3 is a Python library to write CZML, a JSON format for describing
 a time-dynamic graphical scene, primarily for display in a web browser running
 Cesium.")
-    (license license:expat)))
+    (license license:bsd-3)))
 
 (define-public python-drizzle
   (package
-- 
2.47.1

